recharts-bar-chart/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I'm pretty sure we can just get rid of this, right @xyin96 ?

Reply:
yeah i'm not even sure where this is referenced.